src/app/utils/test-model.utils.ts

```diff
@@ -36,3 +38,23 @@ function mockTestType(args?: Partial<TestType>): TestType {
 
   return { ...mock, ...args };
 }
+
+function mockTestTypeCategory(args?: Partial<TestTypeCategory>): TestTypeCategory {
+  const mock: TestTypeCategory = {
+    id: '1',
+    name: 'Annual test',
+    linkedIds: ['38', '39'],
+    testTypeName: 'Annual test',
+    forVehicleType: ['psv'],
+    forVehicleSize: ['small', 'large'],
+    forVehicleConfiguration: ['articulated', 'rigid'],
+    forVehicleAxles: null,
+    forEuVehicleCategory: null,
+    forVehicleClass: null,
+    forVehicleSubclass: null,
+    forVehicleWheels: null,
```

Comment:
why do we put some meaningful default value here. It compliments the mock object. and in the future we would not change it